src/models/methods/subscribe.ts

```diff
@@ -0,0 +1,107 @@
+import { BaseRequest, BaseResponse } from './baseMethod'
+import { Currency, StreamType } from "../common"
+import { TransactionMetadata } from '../common/transaction'
+
+interface Book {
+    taker_gets: Currency
+    taker_pays: Currency
+    taker: string
+    snapshot?: boolean
+    both?: boolean
+}
+  
+export interface SubscribeRequest extends BaseRequest {
+    command: "subscribe"
+    streams?: StreamType[]
+    accounts?: string[]
+    accounts_proposed?: string[]
+    books?: Book[]
+    url?: string
+    url_username?: string
+    url_password?: string
+}
+
+export interface SubscribeResponse extends BaseResponse {
+    result: any
+}
+
+interface BaseStream {
+    type: string
+}
+
+export interface LedgerStream extends BaseStream {
```

Comment:
I put streams here, I couldn't really figure out a better place for them but I can move them if you think it would be better suited elsewhere